src_assets/common/assets/web/config.html

```diff
@@ -947,19 +947,18 @@ <h2 class="accordion-header">
                   </div>
                 </div>
 
-                <!-- Prefer lower encoding latency over increased power consumption -->
+                <!-- Prevent GPU downclocking -->
                 <div class="mb-3" v-if="platform === 'windows'">
-                  <label for="nvenc_latency_over_power" class="form-label">Prefer lower encoding latency over increased power
-                    consumption</label>
+                  <label for="nvenc_latency_over_power" class="form-label">Prevent GPU downclocking</label>
                   <select id="nvenc_latency_over_power" class="form-select" v-model="config.nvenc_latency_over_power">
                     <option value="disabled">Disabled</option>
                     <option value="enabled">Enabled (default)</option>
                   </select>
                   <div class="form-text">
-                    Adaptive P-State algorithm which NVIDIA drivers employ doesn't work well with low latency streaming, so
-                    sunshine requests high power mode explicitly.<br>
-                    Disabling it is not recommended since this can lead to
-                    <strong>significantly increased encoding latency</strong>.
+                    When set to enabled, this option prevents NVIDIA GPUs from downclocking via adaptive P-State while a Sunshine stream is active.<br>
+                    This may lead to decreased encoding latency, but it will also render a major GPU power efficiency feature inoperative.<br>
+                    As a consequence, the GPU's temperature as well as power and fan usage will greatly increase.<br>
+                    This may lower the GPU's durability and increase your energy bills and likelihood of computer issues due to overheating.
```

Comment:
> This may lower the GPU's durability and increase your energy bills and likelihood of computer issues due to overheating.

Nvidia makes no such claims on the option to "Prefer maximum performance" in the official Control Panel. They only state increased power consumption.

![image](https://github.com/LizardByte/Sunshine/assets/2695644/49c982f0-0bae-4eb7-aef2-00334d5c2f29)

There's no reason a non-defective GPU should fail by simply idling at the nominal GPU clock, nor does any Nvidia documentation I have found indicate this would be the case. Nvidia provides the option to _globally_ configure maximum performance mode (which would run 24/7 regardless of application), and that option would be quite dangerous if hardware damage could occur.

And for our usage, the P-State change is only happening _while Sunshine is actively streaming_. If you're leaving Moonlight connected 24/7, well you're going to have plenty of other power being wasted by doing that (keeping the client and host displays awake all the time, for one). The difference in power consumption reported by GPU-Z on my RTX 2080 between idle at the desktop and Sunshine streaming with "Prefer maximum performance" while idle at the desktop is only 30W, which is not something that's going to stress the cooling system of a GPU or gaming system at all. The card has to handle 5-10x that while gaming. My card even still operates completely passively while Sunshine is connected in maximum performance mode.

Sure, it's using more power with Maximum Performance mode and maybe some cards use more power while idle than others, but that's why this option exists for the user to select whether they want that tradeoff.